src/triggers/comment.ts

```diff
@@ -18,13 +25,49 @@ interface CommentsResponse {
             id: string;
             name: string;
           };
-        };
+        } | null;
+        projectUpdate: {
+          id: string;
+          body: string;
+          user: {
+            id: string;
+            name: string;
+            email: string;
+            avatarUrl: string;
+          };
+          url: string;
+          project: {
+            id: string;
+            name: string;
+            url: string;
+          };
+        } | null;
+        documentContent: {
+          id: string;
+          content: string;
+          project: {
+            id: string;
+            name: string;
+            url: string;
+          };
+        } | null;
```

Comment:
Should we leave this one out for now? As DocumentContent could be related to other entities and not just projects.

Reply:
I think we should include but I've fixed the query so it fetches `document` first and all documents belong to a project now.